themes/default/content/docs/pulumi-cloud/oidc/client/_index.md

```diff
@@ -0,0 +1,140 @@
+---
+title_tag: OpenID Connect Client Integration for Pulumi
+meta_desc: This page provides an overview of how to configure OpenID Connect integration between
+           third party OIDC Providers and Pulumi.
+title: OpenID Client
+h1: OpenID Connect integration
+meta_image: /images/docs/meta-images/docs-meta.png
+menu:
+    pulumicloud:
+        parent: openid-connect
+        identifier: openid-connect-client
+        weight: 4
+---
+
+Pulumi supports establishing trust relationships with third party OIDC providers by leveraging id_tokens and allowing it to be exchanged for a short-lived Pulumi access token. This mechanism enhances security by eliminating the necessity for hardcoded credentials.
+
+## Overview
+
+For third party services that have capabilities to issue OIDC id_tokens, it is possible to register them as a trusted OICD Issuer to leverage these tokens to be exchanged by a short-lived Pulumi access token automatically to avoid having to store hardcoded credentials.
```

Comment:
```suggestion
For third party services that have capabilities to issue OIDC id_tokens, it is possible to register them as a trusted OIDC Issuer to leverage these tokens to be exchanged by a short-lived Pulumi access token automatically to avoid having to store hardcoded credentials.
```